src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/NovelAdjacencyReferenceLocations.java

```diff
@@ -52,23 +57,64 @@ protected NovelAdjacencyReferenceLocations(final Kryo kryo, final Input input) {
+        if (sameChromosome) {
+
+            leftBreakpointRefContig = rightBreakpointRefContig = ca.regionWithLowerCoordOnContig.referenceSpan.getContig();
+            final AlignmentInterval one = ca.regionWithLowerCoordOnContig,
+                                    two = ca.regionWithHigherCoordOnContig;
+            final List<AlignmentInterval> deOverlappedTempAlignments = ContigAlignmentsModifier.removeOverlap(one, two, null);
+            final AlignmentInterval deOverlappedOne = deOverlappedTempAlignments.get(0),
+                                    deOverlappedTwo = deOverlappedTempAlignments.get(1);
+            final boolean isLikelyTranslocation =
+                    ca.strandSwitch == StrandSwitch.NO_SWITCH
+                            &&
+                            deOverlappedOne.referenceSpan.getStart() > deOverlappedTwo.referenceSpan.getStart() == deOverlappedOne.forwardStrand;
+
+            if (isLikelyTranslocation) {
+                if (ca.isForwardStrandRepresentation) {
+                    leftBreakpointCoord  = ca.regionWithHigherCoordOnContig.referenceSpan.getStart();
+                    rightBreakpointCoord = ca.regionWithLowerCoordOnContig.referenceSpan.getEnd() - homologyLen;
+                } else {
+                    leftBreakpointCoord  = ca.regionWithLowerCoordOnContig.referenceSpan.getStart();
+                    rightBreakpointCoord = ca.regionWithHigherCoordOnContig.referenceSpan.getEnd() - homologyLen;
+                }
+            } else if (complication.hasDuplicationAnnotation()) {
+                if (complication.hasDupSeqButNoStrandSwitch()) { // simple duplication not involving inverted repeats
+                    final SimpleInterval leftReferenceInterval, rightReferenceInterval;
+                    if (ca.isForwardStrandRepresentation) {
+                        leftReferenceInterval  = ca.regionWithLowerCoordOnContig.referenceSpan;
+                        rightReferenceInterval = ca.regionWithHigherCoordOnContig.referenceSpan;
+                    } else {
+                        leftReferenceInterval  = ca.regionWithHigherCoordOnContig.referenceSpan;
+                        rightReferenceInterval = ca.regionWithLowerCoordOnContig.referenceSpan;
+                    }
+                    if (complication.getDupSeqRepeatNumOnCtg() > complication.getDupSeqRepeatNumOnRef()) {
+                        leftBreakpointCoord = leftReferenceInterval.getEnd() - homologyLen
+                                - (complication.getDupSeqRepeatNumOnCtg() - complication.getDupSeqRepeatNumOnRef()) * complication.getDupSeqRepeatUnitRefSpan().size();
+                    } else {
+                        leftBreakpointCoord = leftReferenceInterval.getEnd() - homologyLen;
+                    }
+                    rightBreakpointCoord = rightReferenceInterval.getStart() - 1;
+                } else {
+                    leftBreakpointCoord     = complication.getDupSeqRepeatUnitRefSpan().getStart() - 1;
```

Comment:
Because for inverted duplications, the ones we can currently catch is not as complicated as the tandem duplication (remember those heuristic optimization we need to do?). And for inverted duplications, there's no contraction here.